sdk/eventgrid/Azure.Messaging.EventGrid/api/Azure.Messaging.EventGrid.netstandard2.0.cs

```diff
@@ -14,11 +14,14 @@ public partial class CloudEvent
         public string DataSchema { get { throw null; } set { } }
         public System.Collections.Generic.Dictionary<string, object> ExtensionAttributes { get { throw null; } }
         public string Id { get { throw null; } set { } }
+        public bool IsSystemEvent { get { throw null; } }
```

Comment:
This is a great idea.

Reply:
I was debating back and forth between this and adding a `bool TryGetAsSystemEventData(out object systemEventData)`.

Reply:
"TryGetAsSystemEventData(out object systemEventData)" isn't as intuitive as IsSystemEvent. I prefer IsSystemEvent too.